### DISCUSSION TITLE
Issues while updating quantities in Amazon

### DISCUSSION THREAD

**Discussion by cihance** [amzn/selling-partner-api-models#3455] [General]
Hello,

Altough, we dont have much product in our inventory, (we need to update 10-20 products in an hour) we have been stugling to find reliable way to send actual quantities to Amazon.

So far i tried feeds api, Listing Api (patching quantity) . For feeds api i tried the feed types (JSON_LISTINGS_FEED, POST_FLAT_FILE_PRICEANDQUANTITYONLY_UPDATE_DATA, POST_INVENTORY_AVAILABILITY_DATA)

We send the actual quantities to amazon and then check after 50 min and see some quantity updates got ignored.  We send them again without knowing if they will be processed or not and wait to check again.

What is the way you use or advise, do you face such problems ? 

Ps. i contacted amazon just recetnly and they asked submissionid timestamp etc.. which i will provide. But investingating a certain submission to understand whats wrong...  i dont think it will be helpfull to resolve it once and for all.

**Comment by tomaszhanc** [comment]
I have been using `_POST_INVENTORY_AVAILABILITY_DATA_` for years (before SP API with MWS API) and it works. Have you checked the feed processing report? Maybe there are some validation errors? This API is asynchronous so processing of the feed may fail. You can find out more here: https://developer-docs.amazon.com/sp-api/docs/feeds-api-v2021-06-30-use-case-guide#step-7-download-the-feed-processing-report

**Comment by cihance** [comment]
i have been checking feed processing reports but they have never been helpful as they dont even return error for non-existing SKUs.
